### PR TITLE
Response object not experimental

### LINKED ISSUE #10384: Issue with "Response.statusText": (short summary here please)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText

#### What information was incorrect, unhelpful, or incomplete?

Doesn't say what the possible values are or even whether they are upper case, title case, etc. I don't think it's necessary to put every possible value, but listing the most common examples (e.g., "OK", "Bad Request", "Unauthorized", "Forbidden", "Not Found", "Request Timeout", "Conflict", and "Internal Server Error") and providing a link to a page describing all the standard status codes would be very helpful.

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText#value

#### What did you expect to see?

List of possible values or several examples

#### Did you test this? If so, how?

N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/response/statustext`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/response/statustext/index.md
* Last commit: https://github.com/mdn/content/commit/66d13347b8cf840d1be7a64417920a5a4802b939
* Document last modified: 2021-10-16T17:32:30.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #10384

Specifically it adds a link to HTTP status codes list to the Response.status and Response.statusText.

While I was here I fixed up all the response items that were tagged as "Experimental" and fixed up the Syntax/values sections and layout across the `Response` object.

### CODE REVIEW

File: files/en-us/web/api/response/redirect/index.md [hunk 1]
```diff
@@ -39,10 +37,10 @@ A {{domxref("Response")}} object.
 
 ### Exceptions
 
-| Exception    | Explanation                                    |
-| ------------ | ---------------------------------------------- |
-| `RangeError` | The specified status is not a redirect status. |
-| `TypeError`  | The specified URL is invalid.                  |
+- `RangeError`
```
Comment (teoli2003) [inline files/en-us/web/api/response/redirect/index.md:40]: As this is not a `DOMException`, we should link to its description.
```suggestion
- {{jsxref("RangeError"}}
```

File: files/en-us/web/api/response/redirect/index.md [hunk 2]
```diff
@@ -39,10 +37,10 @@ A {{domxref("Response")}} object.
...
+  - : The specified status is not a redirect status.
+- `TypeError`
```
Comment (teoli2003) [inline files/en-us/web/api/response/redirect/index.md:42]: As this is not a `DOMException`, we should link to its description.
```suggestion
- {{jsxref("TypeError")}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Looks good to me. Just two exceptions to link to their reference pages.

**Comment by hamishwillee** [comment]
Thanks very much @teoli2003 ! I accepted those two.

File: files/en-us/web/api/response/redirect/index.md [hunk 3]
```diff
@@ -39,10 +37,10 @@ A {{domxref("Response")}} object.
 
 ### Exceptions
 
-| Exception    | Explanation                                    |
-| ------------ | ---------------------------------------------- |
-| `RangeError` | The specified status is not a redirect status. |
-| `TypeError`  | The specified URL is invalid.                  |
+- {{jsxref("RangeError"}}
```
Comment (teoli2003) [inline files/en-us/web/api/response/redirect/index.md:40]: Sorry, I did a typo.
```suggestion
- {{jsxref("RangeError")}}
```

**Review by teoli2003 — COMMENTED** [review]
Sorry I did a typo.

**Comment by hamishwillee** [comment]
And I accepted it. Let us never speak of this again :-)

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**